hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -138,6 +165,32 @@ def hpat_pandas_series_iloc_impl(self):
     return hpat_pandas_series_iloc_impl
 
 
+@overload_attribute(SeriesType, 'index')
+def hpat_pandas_series_index(self):
+    """
+    Pandas Series attribute 'index' implementation.
+        https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.index.html#pandas.Series.index
+
+    Algorithm: result = series.index
+    Where:
+        series: pandas.series
```

Comment:
In new version of the project documentation we have a bit diffrent style of this.